src/config.rs

```diff
@@ -178,6 +178,13 @@ pub struct Config {
         env = "DISCORD_WEBHOOK"
     )]
     pub discord_webhook: Option<String>,
+    #[clap(
+        long,
+        value_name = "PROMETHEUS_METRICS",
+        help = "Toggle to turn Prometheus metrics on/off (off by default)",
```

Comment:
Maybe add a description for which port the server would be at? 

I think maybe even better is for it to be an optional integer, if they provide it then turn on server at that port, otherwise no Prometheus? 

This can wait if we want to get the pr in quick